utils/nctl/sh/scenarios/chainspecs/bond_its.chainspec.toml.in

```diff
@@ -0,0 +1,209 @@
+[protocol]
```

Comment:
likewise here, can you let me know what changes you had to make here that the default cant be used? Just so i have a note to reference if I need to debug this

Reply:
`maximum_round_exponent = 14` and `block_max_transfer_count = 1` are the two differences from the regular chainspec.